src/resources/appointments.js

```diff
@@ -18,6 +18,9 @@ export const appointments = createResourceDescription(
       'initiated_by',
     ],
     filters: ['state', 'start', 'end'],
+    headers: {
+      recaptcha: value => ({ 'X-Im-Not-A-Robot': value }),
+    },
```

Comment:
The reason that I made it a function is so that we have the full flexibility.
We can now do the following:

1. Map to multiple headers
2. Format the header value